HW20/VictoryPolicy.cpp

```diff
@@ -0,0 +1,18 @@
+#include "VictoryPolicy.h"
+
+class ItemDeck;
+class ModifierDeck;
+class Munchkin;
+
+void VictoryPolicy::apply(Munchkin* munchkin, ItemDeck& itemDeck, ModifierDeck& modifierDeck)
+{
+	munchkin->updateLevelBy(m_levelsCount);
+	munchkin->addItems(itemDeck.generateItems(m_itemsCount));
+	munchkin->addModifiers(modifierDeck.generateModifiers(m_modifiersCount));
+}
```

Comment:
It's not flexible as all different victories are placed in one class. It would be hard to extend this logic without possibly breaking it